src/ml_board/frontend/Dashboard/src/components/graphs/Graph.tsx

```diff
@@ -43,8 +46,58 @@ export default function Graph({ chart_id, exp_id, exp_data }: { chart_id: string
             })),
     };
 
+    const legendOnClickHandler = function (event:any, legendItem:any, legend:any) {
+        let index = legendItem.datasetIndex;
+        let ci = legend.chart;
+        let meta = ci.getDatasetMeta(index);
+
+        // Get the indexes of datasets with hidden === false
+        let visibleIndexes = ci.data.datasets.reduce(function(acc:any, dataset:any, i:any) {
```

Comment:
making a variable a **const** is not just better for developer not to overwrite it later but also better for the reviewer, I know then that this variable is always going to point to the same object or instance in memory generally speaking!

and that's everywhere where not just here 🥲